specification-discussions/20200309-dx-mosaic-testnet-gen1.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "DX for Mosaic testnet gen-1"
+disqus: https://hackmd.io/gTarrQN9QfusSvZeZp2W_w
+---
+
+# DX for Mosaic testnet gen-1
+
+| Version | Last Updated | Component                   |
+| ------- | ------------ | --------------------------- |
+| 0.14    | 09/03/2020   | DX for Mosaic testnet gen-1 |
+
+## User stories
+
+1. As a developer, I should be able to see which metachains exist for goerli (as origin chain)
+2. As a developer, I should be able to see what the base token is for each metachain (Ost or GoEth)
+3. As a developer, I should be able to connect to a public web3 RPC for the origin chain
+4. As a developer, I should be able to connect to a public web3 RPC for a metachain
+5. As a developer, I should be able to get the base token on the origin (GoEth; from faucet)
+6. As a developer, I should be able to get the base token on a metachain (Ost or GoEth; from faucet)
+7. As a developer, I should be able to get Ost ERC20 on the origin chain (from faucet)
+8. As a developer, I should be able to view total deposited ERC20 balances (for all ERC20's for all metachains)
+9. As a user, I should be able to deposit wGoEth, Ost, or any ERC20 into an ERC20gateway on, and receive it as an ERC20 on the metachain (handled by any facilitator)
```

Comment:
`..., except the base token of said metachain`

Reply:
done